shoplist/src/main/java/com/netguru/codereview/ui/model/ShopList.kt

```diff
@@ -0,0 +1,11 @@
+package com.netguru.codereview.ui.model
+
+import com.netguru.codereview.network.model.ShopListItemResponse
+
+class ShopList(
+    val id: String,
+    val userId: Int,
+    val listName: String,
+    val iconUrl: String,
+    val items: List<ShopListItemResponse>
```

Comment:
this parameter is not used, why its inside the model. please follow YAGNI principle.